### PR TITLE
Add failing test

### PR DESCRIPTION
**Pull request by abtris**

### CODE REVIEW

**Comment by abtris** [comment]
@w-vi I make proposal how we can handle this in drain, can you look if is ok?

File: server_test.go [hunk 1]
```diff
@@ -41,6 +41,15 @@ var fullTests = []struct {
 			"heroku.dyno.web:3.000000|g",
 		},
 	},
+	{
+		cnt: 1,
```
Comment (w-vi) [inline server_test.go]: `cnt` should be 3 as we expect 3 data points added to datadog

**Comment by w-vi** [comment]
@abtris apart form the `cnt` it looks good.

**Comment by abtris** [comment]
Why test not failed and make only timeout?

**Comment by w-vi** [comment]
Because now these messages are not processed at all so it waits to recieve 3 statsd messages which will never come, the log line is dropped

**Pull request closed without merging**